### PR TITLE
Configurable logout redirect url via LOGINAS_LOGOUT_REDIRECT_URL

### PR DESCRIPTION
**Pull request by intellisense**
When admin logging out from delegated session we might want to take user back to admin pages this now can be accomplished by setting var `LOGINAS_LOGOUT_REDIRECT_URL` instead of always redirecting user back to `settings.LOGIN_REDIRECT_URL`

```
# settings.py

from django.core.urlresolvers import reverse_lazy
LOGINAS_LOGOUT_REDIRECT_URL = reverse_lazy('admin:index')
```

### CODE REVIEW

**Comment by intellisense** [comment]
Hi,

I have created three pull request #46 #47 and this one. I have created them separately so that you can decide which one is important for merging. Though I am afraid merging one will raise conflicts in other. Better is to review all three of them and if you think all are good then let me know and I will create a single pull request containing all of them from my master branch.

Kind Regards!

File: README.md [hunk 1]
```diff
@@ -79,6 +79,15 @@ from django.core.urlresolvers import reverse_lazy
 LOGOUT_URL = reverse_lazy('loginas-logout')
 ```
 
+Additionally you can also specify the redirect url for logout default is `settings.LOGIN_REDIRECT_URL`
```
Comment (skorokithakis) [inline README.md]: Can you clean up the wording a bit? Like so:

Additionally, you can specify the redirect url for logout (the default is `settings.LOGIN_REDIRECT_URL`).

Comment (intellisense) [inline README.md]: Updated in commit [f0cb3f6
](https://github.com/skorokithakis/django-loginas/pull/49/commits/f0cb3f6261063f36bf791a1ff1b42fbf7757fe36) Thanks!

**Comment by skorokithakis** [comment]
Thank you for your help!

**Comment by MounirMesselmeni** [comment]
When do you plan to release this?

**Comment by skorokithakis** [comment]
Released 0.3.1, thanks!

**Comment by benjaoming** [comment]
Thanks!

**Pull request merged**